docs/notebooks/zero_inflated_regression.ipynb

```diff
@@ -0,0 +1,1905 @@
+{
```

Comment:
<blockquote>However, notice how the hurdle model uses the Poisson cumulative distribution function to evaluate the probability of a zero. This can be thought of as the "zero mass" component</blockquote><p>This is correct, but I'm not sure it's clear. I would add that this evaluation of of a Poisson CDF at zero, but that doesn't play into the P(Y=0) but into the P(Y = y) where y &gt; 0. You explain it correctly in the final sentence.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/725/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>